bolt: Disable the check for /proc/self/map_files. Might not be available on build machines

--- a/bolt/CMakeLists.txt
+++ b/bolt/CMakeLists.txt
@@ -40,16 +40,6 @@ if ((CMAKE_SYSTEM_PROCESSOR STREQUAL "x8
   set(BOLT_ENABLE_RUNTIME_default ON)
 endif()
 option(BOLT_ENABLE_RUNTIME "Enable BOLT runtime" ${BOLT_ENABLE_RUNTIME_default})
-if (BOLT_ENABLE_RUNTIME)
-  # Some systems prevent reading /proc/self/map_files
-  execute_process(COMMAND ls /proc/self/map_files
-    RESULT_VARIABLE LS OUTPUT_QUIET ERROR_QUIET)
-  if (LS)
-    set(BOLT_ENABLE_RUNTIME OFF)
-    message(WARNING
-      "BOLT runtime is disabled as /proc/self/map_files is unreadable.")
-  endif()
-endif()
 
 set(BOLT_CLANG_EXE "" CACHE FILEPATH "Path to clang executable for the target \
 architecture for use in BOLT tests")
